Stripe/STPAddCardViewController.m

```diff
@@ -120,12 +98,8 @@ - (void)createAndSetupViews {
     cardImageView.frame = CGRectMake(0, 0, self.view.bounds.size.width, cardImageView.bounds.size.height + (57 * 2));
     self.cardImageView = cardImageView;
     self.tableView.tableHeaderView = cardImageView;
-    self.emailCell = [[STPRememberMeEmailCell alloc] initWithDelegate:self];
-    if ([STPEmailAddressValidator stringIsValidEmailAddress:self.prefilledInformation.email]) {
-        self.emailCell.contents = self.prefilledInformation.email;
```

Comment:
Hmm, actually I dunno, it seems potentially useful for people to have the ability to prefill this stuff in shipping address form?